src/components/sheet/sheet-action-buttons/WithdrawActionButton.js

```diff
@@ -0,0 +1,34 @@
+import analytics from '@segment/analytics-react-native';
+import React, { useCallback, useState } from 'react';
+import { colors } from '../../../styles';
+import { neverRerender } from '../../../utils';
+import { ComingSoonFloatingEmojis } from '../../floating-emojis';
+import SheetActionButton from './SheetActionButton';
+
+function WithdrawActionButton({ color = colors.white, symbol, ...props }) {
+  const [didTrack, setDidTrack] = useState(false);
+
+  const handlePress = useCallback(() => {
+    if (!didTrack) {
+      analytics.track('Tapped placeholder Withdraw button', {
+        category: 'liquidity pool',
+        label: symbol,
+      });
+      setDidTrack(true);
+    }
+  }, [didTrack, setDidTrack, symbol]);
```

Comment:
suuuper minor - can remove setDidTrack from dependency list

Reply:
tru

Reply:
actually i dont care, this code is temporary anyways